docs/transaction-api.asciidoc

```diff
@@ -243,3 +243,29 @@ If the transaction has already ended,
 nothing happens.
 
 Alternatively you can call <<apm-end-transaction,`apm.endTransaction()`>> to end the active transaction.
+
+[[transaction-outcome]]
+==== `transaction.outcome`
+
+[small]#Added in: v3.12.0#
+
+The Node.js agent automatically sets an `outcome` property on transactions.  This property will be one of three values:
+
+- `success`: Indicates the transaction's operation was a success.
+
+- `failure`: Indicates the transaction's operation was _not_ a success.
+
+- `unknown`: Indicates we were unable to determine if the transaction's operation was a success or not.  An `unknown` outcome removes a transaction from error rate considerations.
+
+A transaction is considered a success if the underlying HTTP request handling produces a response with a status code that is less than `500`.  A status code of `500` or greater is considered a failure.
+
+Non-HTTP transactions will begin with an outcome of `unknown`.
+
+[[transaction-setoutcome]]
+==== `transaction.setOutcome(outcome)`
+
+[small]#Added in: v3.12.0#
+
+* `outcome` +{type-string}+
+
+The `setOutcome` method allows an end user to override the Node.js agent's default setting of a transaction's `outcome` property.  The `setOutcome` method accepts a string of either `success`, `failure`, or `unknown`, and will force the agent to report this value for a specific's span.
```

Comment:
I think you forgot to update this one

Reply:
You can blame it on the ❄️/🧊

Reply:
Strange -- I accepted the GitHub suggested edit you made but  must have overridden it with a local copy. Change made: https://github.com/elastic/apm-agent-nodejs/commit/f09ed4bf5525ba63de117cde9121b608698bc22b#diff-4d36a86458463b6e2f5fe79f60f0c135455c5ac4e635b92dcd13f92e4eeb342e

Reply:
Oh! Strange indeed.